common/schema/DHQauthor-TEI.xml

```diff
@@ -1707,9 +1726,17 @@ email (should only have an email address)
       <elementSpec ident="note" mode="change" module="core">
         <classes mode="change">
           <memberOf key="model.common" mode="add"/>
+          <memberOf key="model.imprintPart" mode="add"/>
```

Comment:
Given that in DHQ `<note>` is not a member of model.global, and thus is not already in the content model of `<imprint>`, I think this is the right way to do this. But my instinct is that this information would be better served by a `<ptr>` (or perhaps `<idno>`) that is a direct child of `<monogr>`, instead. That allows the URL itself to be on the `@target` attribute, and thus be checked for syntactic correctness (although to be fair, that does not say much), and allows a `@type` to classify what the pointer is to. Furthermore it is what TEI-in-Libraries recommends as a best practice.